lib/checker/mochikit.coffee

```diff
@@ -100,8 +97,7 @@ module.exports = (stats, contents) ->
 
   results = []
   for line, i in lines
-    for regex, i in mochikitRegexes
+    for regex, r in mochikitRegexes
       if regex.test(line)
-        results.push(line: i, level: 'error', message: "Usage of function #{mochikitFunctions[i]})")
-
+        results.push(line: i + 1, level: 'error', message: "Usage of function #{mochikitFunctions[r]})")
   results
```

Comment:
line number was being reset to the index of the regex

+1 because the count starts from 0